12LessonTwelve/homework/index.html

```diff
@@ -20,7 +20,7 @@ <h1 class="section-title">Best Selling Products</h1>
         </div>
     </header>
     <section class="products">
-        <div class="col-lg-3 col-sm-6 col-xs-12 text-center patient">
+        <div class="col-lg-10 col-sm-8 col-xs-12 text-center patient">
             <article>
                 <figure>
                     <a href="#">
```

Comment:
Ethan,

 

I have corrected the site.  For some reason I changed a parameter that should not have been changed.  Please lte me know if you have any questions on this but I believe it is there.

 

Thank You,

 

Steve

 

From: Ethan Stenis [mailto:notifications@github.com] 
Sent: Tuesday, September 13, 2016 5:56 PM
To: AustinCodingAcademy/intro-workbook <intro-workbook@noreply.github.com>
Cc: Steve Anderson <anderson-steven@att.net>; Author <author@noreply.github.com>
Subject: Re: [AustinCodingAcademy/intro-workbook] Austin.Intro.2.Anderson.Steve (#84)

 

In 12LessonTwelve/homework/index.html <https://github.com/AustinCodingAcademy/intro-workbook/pull/84#discussion_r78659978> :

> @@ -20,7 +20,7 @@ <h1 class="section-title">Best Selling Products</h1>
>          </div>
>      </header>
>      <section class="products">
> -        <div class="col-lg-3 col-sm-6 col-xs-12 text-center patient">
> +        <div class="col-lg-10 col-sm-8 col-xs-12 text-center patient">
>              <article>
>                  <figure>
>                      <a href="#">

Something funky is going here with that first div. Please fix and I can bump your grade up.
 <https://cloud.githubusercontent.com/assets/12698279/18494352/3a89f2c0-79db-11e6-883f-c899103b9e19.png> 

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub <https://github.com/AustinCodingAcademy/intro-workbook/pull/84/files/279afa322304adf9e4b6c376140c121144898f6b..44477f170625943328e0571dc10b9611aac8ad6b#r78659978> , or mute the thread <https://github.com/notifications/unsubscribe-auth/ATJFMK_MNiIl6Bfl9TcD1F1A5hC4Sop9ks5qpyntgaJpZM4JVvJX> .  <https://github.com/notifications/beacon/ATJFMEE2y7b67uMYFqtAZx-QW6Tl8Lm1ks5qpyntgaJpZM4JVvJX.gif>